fenicsprecice/fenicsprecice.py

```diff
@@ -251,9 +249,6 @@ def write_data(self, write_function):
         assert (self._write_function_type == determine_function_type(w_func, self._fenics_dims))
         assert (write_function.function_space() == self._write_function_space)
```

Comment:
Here, we have `assert (...)` (with blank), not `assert(...)` (without blank)

Reply:
Every `assert` is now of the style `assert (...)`.

Reply:
All fine. As long as it is consistent :)